modules/release-notes/pages/0.6.24-rn.adoc

```diff
@@ -21,14 +21,44 @@ The most significant new features and capabilities include updates in the follow
 
 === DFX
 
-* A new `+deploy-wallet+` subcommand enables you to specify the canister identifier for your cycles wallet WebAssembly module (WASM).
+* With this release, you must have a **wallet canister** with **cycles** to deploy or manage applications on the {IC}.
++
+For local development, `+dfx+` automatically creates a wallet for you when you run `+dfx canister create+` or `+dfx deploy+` commands within each project.
++
+To deploy and manage applications on the main {IC} network, you must provide the {company-id} Foundation with a principal associated with your user identity and receive a **wallet canister identifier** in return.
++
+To deploy applications on the {IC} network:
++
+--
+. Download and install the {sdk-short-name}.
+.  Run the `dfx identity get-principal` command to create your default identity and principal:
++
+[source,bash]
+----
+dfx identity get-principal
+----
++
+Running the command displays output similar to the following:
 +
-For example, to deploy a cycles wallet for local development, you can run a command similar to the following:
 ....
-dfx identity deploy-wallet <canister-id>
+Creating the "default" identity.
+  - generating new key at /Users/lisagunn/.config/dfx/identity/default/identity.pem
+Created the "default" identity.
+wcp5u-pietp-k5jz4-sdaaz-g3x4l-zjzxa-lxnly-fp2mk-j3j77-25qat-pqe
 ....
+. Submit the principal to get your wallet canister identifier.
+. Run the `+dfx identity set-wallet+` command to associate your wallet canister identifier with your default identity.
++
+[source,bash]
+----
+dfx identity set-wallet <wallet-canister-identifier>
+----
+. Open the wallet application in a web browser by navigating to the canister with a URL similar to the following:
 +
-Keep in mind that the wallet canister identifier is specific to an active identity—for example, your `+default+` identity—on a specific network.
+....
+https://<canisterId>.ic0.app/?host=https://mercury.dfinity.network
```

Comment:
Not sure if this is ready to be in the public release notes just yet. Maybe we should include it once we are ready to onboard the general public? (the people who gave us their principals for sodium)